tutorials/plot_visualize_evoked.py

```diff
@@ -99,13 +99,14 @@
 # more information.
 #
 # We can also combine the two kinds of plots to on using the ``plot_joint``
-# method of Evoked objects. Notice the ``topomap_args`` and ``ts_args``
-# parameters of :func:`mne.Evoked.plot_joint`.
-# You can pass key-value pairs as a python dictionary that gets passed as
-# parameters to the topomaps and time series of the joint plot.
-evoked_r_aud.plot_joint()
-
-# For slightly more elaborate plots, use these ``topomap_args`` and ``ts_args``
+# method of Evoked objects. Called as-is (``evoked.plot_joint()``), this
+# function should give a stylish and informative display of spatio-temporal
```

Comment:
I personally am all style over substance, but I understand that others might disagree.